### PR TITLE
Mobile-menu

### PR DESCRIPTION
**Pull request by sergiogval**
This feature delivers a classic mobile menu that shows up when the user tap/click the burger menu at the upper right cortner.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,32 +1,173 @@
...
+  padding: 1vh 2.7em;
+  justify-content: center;
+  align-items: end;
+  height: 88vh;
+  width: 100%;
+  display: none;
+  position: absolute;
+  opacity: 1;
+  top: 9vh;
+}
```
Comment (rotimiazeez) [inline css/style.css]: - [ ] Kindly make sure your design matches your preferred template design😉
- Your Design
 screenshot (11)
- Figma design
screenshot (10)

Comment (rotimiazeez) [reply css/style.css]: - [x] Kindly make the menu fixed so it won't scroll like the rest of the page and please make sure when user clicks any option of the mobile menu, the mobile menu disappear. In your case in scroll down to the page but when you scroll up the mobile menu is still visible✌

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM👋, 
## Good job so far!✌
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (sergiogval) [reply css/style.css]: Done

**Review by ignatius22 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**